certified-connectors/HVI Vehicle Inspection/apiDefinition.swagger.json

```diff
@@ -0,0 +1,458 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "1.0.0",
+    "title": "HVI Vehicle Inspection",
+    "description": "HVI Inspection data per vehicle",

```

Comment:
Please make the description more informative as this will be shown on power automate connector details.